desktop-app/src/pages/NewSimulation.jsx

```
@@ -0,0 +1,630 @@
+        return null;
+    }
+  };
+
+  return (
+    <div className="max-w-4xl mx-auto p-6">
+      {/* Header */}
+      <div className="mb-8">
+        <button
+          onClick={() => navigate('/simulations')}
+          className="flex items-center text-sm text-gray-600 hover:text-gray-900 mb-4"
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)